tests/providers/amazon/aws/sensors/test_batch.py

```diff
@@ -42,7 +42,7 @@ def setup_method(self):
     @mock.patch.object(BatchClientHook, "get_job_description")
     def test_poke_on_success_state(self, mock_get_job_description):
         mock_get_job_description.return_value = {"status": "SUCCEEDED"}
-        assert self.batch_sensor.poke({})
+        assert self.batch_sensor.poke({}) is True
```

Comment:
Small nitpick from @ferruzzi :
https://github.com/apache/airflow/pull/28139#pullrequestreview-1207017465

Reply:
Thanks for including it here.  :+1: